File: elasticsearch-all/flink-java-es/src/main/java/cn/jly/elasticsearch/beans/Person.java
package cn.jly.elasticsearch.beans;

import lombok.Data;
import lombok.NoArgsConstructor;
import scala.compat.java8.collectionImpl.ProxySpliteratorViaNext;

/**
 * @author lanyangji
 * @date 2021/4/18 下午 10:08
 * @packageName cn.jly.elasticsearch.beans
 * @className Person
 */
@Data
@NoArgsConstructor
public class Person {
    private Long id;
    private String name;
    private Integer age;
    private String email;
}

File: elasticsearch-all/spark-streaming-elasticsearch/src/main/scala/cn/jly/spark/es/Person.scala
package cn.jly.spark.es

import com.fasterxml.jackson.databind.ObjectMapper

/**
 * @author lanyangji
 * @date 2021/4/14 下午 8:52
 * @packageName
 * @className cn.jly.spark.es.Person
 */
case class Person(id: Long, name: String, age: Int, email: String) extends Serializable {
}